### PR TITLE
Update metadata & compress_deps

### PR DESCRIPTION
**Pull request by Hzxin**
### Metadata changes
1. Added localization lines (might not be perfect). 
 
2. Remove the `target` path for the all-dependecies.jar path in meta-data. That means it will be placing all-dependencies.jar under `/{bug}/{src}` or `{/{bug}/{src}/{failing_module}`. 

- Vul4j uses clean install to compile which removes the entire `/target` folder.

### Other changes
1.  Added empty line  in `build_subject` for certain bugs where the fault/bugs involves deletion or insertion only.
2. Added compress_deps to copy all dependencies needed into all-dependencies.jar
       - compress_deps will remove extra junit jar from dependencies.

### CODE REVIEW

File: codehaus-plexus/CVE-2017-1000487/patches/tbar/Patch_451_277.txt
Comment (Marti2203) [inline codehaus-plexus/CVE-2017-1000487/patches/tbar/Patch_451_277.txt:1]: This should not be here

File: apache/CVE-2017-5662/Dockerfile-tbar-vul4j-apache-cve-2017-5662
Comment (Marti2203) [inline apache/CVE-2017-5662/Dockerfile-tbar-vul4j-apache-cve-2017-5662:1]: same as this

File: build_subject_maven
Comment (Marti2203) [inline build_subject_maven:1]: reference to tbar?

Comment (Hzxin) [reply build_subject_maven:1]: Have modified. Was used as init test file for TBar. But should not affect the build_script in each subject.

**Pull request merged**